template.yaml

```diff
@@ -333,3 +355,20 @@ Resources:
       RestApiId: !Ref NvaCristinProxyApi
       Stage: !Ref NvaCristinProxyApi.Stage
 
+  NvaCristinProxyInternalApiBasePathMapping:
```

Comment:
Will this crash with NvaCristinProxyBasePathMapping?

Reply:
fixed